NEL.MESH/Services/Foundations/Chunks/ChunkService.Validations.cs

```diff
@@ -13,7 +13,7 @@ private static void ValidateMessageIsNotNull(Message message)
         {
             if (message is null)
             {
-                throw new NullMessageException();
+                throw new NullMessageException(message: "Message is null.");
```

Comment:
Please fix this.  Thi service shoud not use another services exception. Create `NullMessageChunkException` please